metrics/prometheus/prometheus.go

```diff
@@ -134,21 +134,21 @@ func RegisterCallbackGaugeWithLabels(namespace, subsystem, name, help string, co
 	))
 }
 
-type prometheusHistogram struct {
+type prometheusSummary struct {
 	*prometheus.SummaryVec
 	Pairs map[string]string
 }
 
-// NewHistogram returns a new Histogram backed by a Prometheus summary. It
-// uses a 10-second max age for bucketing. The histogram is automatically
-// registered via prometheus.Register.
-func NewHistogram(namespace, subsystem, name, help string, fieldKeys []string) metrics.Histogram {
-	return NewHistogramWithLabels(namespace, subsystem, name, help, fieldKeys, prometheus.Labels{})
+// NewSummary returns a new Histogram backed by a Prometheus summary. It uses
+// a 10-second max age for bucketing, emulating statsd. The histogram is
```

Comment:
Users at SC rarely needed to change the 10m max age. That doesn't prove that they wouldn't have been happier with 10s, though... It's a tough call how many of the config options you want to let leak into an abstracted/unified API like gokit metrics. Going with statsd sounds reasonable to me for the sake of unification. Given that I have created all the options in prometheus.SummaryOptions, I think they are all worthwhile. ;) I have to leave it to your judgement what is desired for gokit. I believe you have the understanding now to come to an informed decision.